spraybot_row_detection/params/detection.yaml

```
@@ -11,6 +11,7 @@ row_detection:
     desired_x: 1.5      # x value of desired goal pose 
     pose_pub_topic: "/goal_pose"
     cluster_sub_topic: "/row_detection_processor_xyz/euclidean_cluster_extraction/clusters"
+    in_row_flag: "/in_row_flag"
```

```suggestion
    detection_status_topic: "/in_row_flag"
```
Maybe this name would make more sense. Since this is indicating whether row detection detected rows or not. We would likely have a separate valuable for checking if we are within the rows.